### ISSUE TITLE
cannot reproduce survey-study.ipynb

### ISSUE THREAD

**Issue by leomiquelutti** [BRGM/razorback#7]
Dear,

I am trying to reproduce [`survey-study.ipynb`](https://github.com/BRGM/razorback/blob/master/docs/source/tutorials/survey-study.ipynb) as in [this EMinar](https://www.youtube.com/watch?v=b75_BZR4D_I). I tried to do it both in [Google Colab](https://colab.research.google.com/drive/1WIcjuVmBEASnOQE1Cie6peTG2JjRczhq?usp=sharing) (open to access to everyone) as well as locally.

And, although `glob` identifies the files, the `inventory` is not filled, as shown in the images below taken from the local execution.

![image](https://user-images.githubusercontent.com/9113903/129193020-724e0992-31bc-4d65-ba21-aedf278d924e.png)
![image](https://user-images.githubusercontent.com/9113903/129193584-09e0e98e-cabd-4ad1-b014-f801244c95d5.png)

There is no error message or alike. Debugging it in VS Code showed me that the code is bypassing the `for` loop.

Can you help me, please?

**Comment by leomiquelutti** [comment]
I solved it changing the `pattern` from  
`pattern = "*\\site{site}\\*\\*_T{channel}_*.ats"`  
to  
`pattern = "*\\*\\*\\*\\*\\*\\site{site}\\*\\*_T{channel}_*.ats"`.

But now I get the error `ValueError: 'MFS07039.txt' file not found in 'metronix_calibration' data directory`. And the file is the folder within the [data for the tutorial](https://github.com/BRGM/razorback-tutorial-data).

**Comment by leomiquelutti** [comment]
I can solve it by changing the variable `METRONIX_DATA_PATH` in `calibrations.py`. And although it can be done in Google Colab, it is not straightforward. Any suggestions?

**Comment by arturbenevides** [comment]
> Dear,
> 
> I am trying to reproduce [`survey-study.ipynb`](https://github.com/BRGM/razorback/blob/master/docs/source/tutorials/survey-study.ipynb) as in [this EMinar](https://www.youtube.com/watch?v=b75_BZR4D_I). I tried to do it both in [Google Colab](https://colab.research.google.com/drive/1WIcjuVmBEASnOQE1Cie6peTG2JjRczhq?usp=sharing) (open to access to everyone) as well as locally.
> 
> And, although `glob` identifies the files, the `inventory` is not filled, as shown in the images below taken from the local execution.
> 
> ![image](https://user-images.githubusercontent.com/9113903/129193020-724e0992-31bc-4d65-ba21-aedf278d924e.png)
> ![image](https://user-images.githubusercontent.com/9113903/129193584-09e0e98e-cabd-4ad1-b014-f801244c95d5.png)
> 
> There is no error message or alike. Debugging it in VS Code showed me that the code is bypassing the `for` loop.
> 
> Can you help me, please?

Léo,
The same happened to me when I was trying to reproduce the tutorial, but I was using the Spyder's IDE and a file in another folder. Then I tried using the same file 'survey-study.ipynb' in the jupyter notebook and it worked. I'm still trying to understand.

**Comment by leomiquelutti** [comment]
Hi @arturbenevides,

thanks for the reply. 

I solved it by adapting the _survey-study.ipynb_ to Google Colab, so now data is downloaded automatically and the calibration files are moved to the proper place within the google server.

I created a Pull Request to add this file into the main `razorback` repo, and it can also be found in [here](https://github.com/leomiquelutti/razorback/blob/master/docs/source/tutorials/survey_study_colab_version.ipynb).
